src/Traits/TransactionTrait.php

```diff
@@ -96,7 +117,7 @@ trait TransactionTrait
      *
      * @Assert\Type(type="string")
      *
-     * @Groups({"create", "delete", "get", "update"})
+     * @Groups({"get"})
```

Comment:
Transaction's orderId and id properties are set by the caller on create

Reply:
fixed